### DISCUSSION TITLE
Default X and B button input reversed when using GC Controller with GBros Adapter

### DISCUSSION THREAD

**Discussion by ApolloVI** [darthcloud/BlueRetro#485] [General]
So I just installed the internal BlueRetro adapter in my Gamecube with the idea that I could pair an 8BitDo GBros adapter and use an original GC controller wirelessly. It works brilliantly with one exception. The default seems to be that X and B are reversed. I think I realize why - BlueRetro is seeing it as an Xinput, which has a layout like an Xbox controller with B on the right and X on the left, so it makes sense to flip X and B inputs, as its more similar to an original GC layout. But as a GC controller using Xinput, you still have those reversed inputs with an original layout. 

I was able to reverse this in the Advance config menu by flipping B and X once again, but I was wondering if this was intentional. Would it be possible to detect a GBros and have the inputs already reversed back by default? Or would that compromise other Xinput layouts for other controllers using GBros and not be worth the change?

**Comment by darthcloud** [comment]
Thianks for reporting. Not sure how I missed that.

Fixed in final v1.8 release
https://github.com/darthcloud/BlueRetro/releases/tag/v1.8

**Reply by ApolloVI** [reply]
Oh awesome! Glad this was of help. Thanks and keep up the great work!
